server/command.go

```diff
@@ -484,3 +485,74 @@ func (p *Plugin) subscribeCommand(parameters []string, channelID string, config
 
 	return fmt.Sprintf("Successfully subscribed to %s.%s", fullPath, hookStatusMessage)
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	gitlabCommand := model.NewAutocompleteData("gitlab", "[command]", "Avaliable commands: connect, disconnect, todo, subscribe, unsubscribe, me, settings, webhook")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your GitLab account")
+	gitlabCommand.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "disconnect your GitLab account")
+	gitlabCommand.AddCommand(disconnect)
+
+	todo := model.NewAutocompleteData("todo", "", "Get a list of unread messages and merge requests awaiting your review")
+	gitlabCommand.AddCommand(todo)
+
+	subscribe := model.NewAutocompleteData("subscribe", "[command]", "Available commands: list")
+	subscribeList := model.NewAutocompleteData("list", "", "List current channel subscritpions")
+	subscribe.AddCommand(subscribeList)
+
+	subscribeChannel := model.NewAutocompleteData("subscribe", "owner[/repo] [features]", "Subscribe the current channel to receive notifications from a project")
```

Comment:
Given that this subcommand has the trigger `subscribe` it will add `subscribe` twice. 
![Screenshot from 2020-06-15 11-17-17](https://user-images.githubusercontent.com/16541325/84649871-8fc67000-af07-11ea-8974-791777ea7b1b.png)
![Screenshot from 2020-06-15 11-17-23](https://user-images.githubusercontent.com/16541325/84649879-92c16080-af07-11ea-9c22-626a8f5df6be.png)

Reply:
I made changes but i think they expose a UI bug.